### PR TITLE
[DSCP-166] [AD-308] Split Ariadne lib into Core and Cardano

### PR DESCRIPTION
**Pull request by goodsoft**
**YT issue:** https://issues.serokell.io/issue/AD-308 / https://issues.serokell.io/issue/DSCP-166

**Checklist:**

- [x] Updated docs if necessary
  - [x] [README](README.md)
  - [x] [TUI usage guide](docs/usage-tui.md)
- [ ] Adressed HLint warnings and hints
- [ ] Rebased branch on current master and squashed all "Address PR comment" commits
- [x] Tested my changes if they modify code

**Description:**
Ariadne.Config and, transitively, Ariadne.Update still depend on Cardano.
Configuration should be modularized in the future as well, and moved into Core.

### CODE REVIEW

File: ariadne/core/src/Ariadne/Knit/Help.hs [hunk 1]
```diff
@@ -28,7 +27,7 @@ commandHelp Knit.CommandProc{..} =
           Knit.OpUnit -> "()"
           Knit.OpAndThen -> "(;)"
         prefixes = name cpName : repeat (T.replicate (T.length $ name cpName) " ")
-        helpLines = map (\l -> "-- " <> l <> "\n") $ leftAlign 40 cpHelp
+        helpLines = map (\l -> "-- " <> l <> "\n") $ T.chunksOf 40 cpHelp
```
[the remark this answers is not in the document]
Comment (gromakovsky) [reply ariadne/core/src/Ariadne/Knit/Help.hs]: Wait, it looks wrong, it doesn't consider spaces.

[the remark this answers is not in the document]
Comment (goodsoft) [reply ariadne/core/src/Ariadne/Knit/Help.hs]: I don't think it's worth pulling the whole library from Cardano, when in the end we'll return commands in some structured way for UI to deal with.

File: ariadne/cardano/import_wallet.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+CARDANO_SL_PATH=${1:-../cardano-sl}
```
[the remark this answers is not in the document]
Comment (goodsoft) [reply ariadne/cardano/import_wallet.sh]: As far as I understand, this script is to be run from `ariadne/` repo root, with `cardano-sl` repo being checked out in the neighboring directory.
So, this one is correct and should stay as is.

File: ariadne/core/src/Ariadne/Knit/Help.hs (hunk 1 above)
[the remark this answers is not in the document]
Comment (gromakovsky) [reply ariadne/core/src/Ariadne/Knit/Help.hs]: Why whole library? You can copy `Pos.Util.Justify` into this file or next to it, can't you?

File: ariadne/cardano/import_wallet.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (gromakovsky) [reply ariadne/cardano/import_wallet.sh]: Hm, okay.

File: ariadne/core/src/Ariadne/Knit/Help.hs (hunk 1 above)
[the remark this answers is not in the document]
Comment (goodsoft) [reply ariadne/core/src/Ariadne/Knit/Help.hs]: ok.

**Pull request merged**